# Publish all joint states -----------------------------------
scout_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Joint velocity controllers ---------------------------------
scout_motor_fr_controller:
  type: velocity_controllers/JointVelocityController
  joint: front_right_wheel
  pid:
    p: 5.0
    i: 20.0
    d: 0.0
    i_clamp: 100.0

scout_motor_fl_controller:
  type: velocity_controllers/JointVelocityController
  joint: front_left_wheel
  pid:
    p: 5.0
    i: 20.0
    d: 0.0
    i_clamp: 100.0

scout_motor_rl_controller:
  type: velocity_controllers/JointVelocityController
  joint: rear_left_wheel
  pid:
    p: 5.0
    i: 20.0
    d: 0.0
    i_clamp: 100.0

scout_motor_rr_controller:
  type: velocity_controllers/JointVelocityController
  joint: rear_right_wheel
  pid:
    p: 5.0
    i: 20.0
    d: 0.0
    i_clamp: 100.0